### PR TITLE
updated-boot-features-documentation

### PR DESCRIPTION
**Pull request by alex-hat-de-rojo**
In `provisioning.adoc` file (in the [Boot Features section](http://karaf.apache.org/manual/latest/#_boot_features)), I added a note about editing boot features in the `features.cfg` file.

### CODE REVIEW

**Comment by alex-hat-de-rojo** [comment]
Ready for review.

File: manual/src/main/asciidoc/user-guide/provisioning.adoc [hunk 1]
```diff
@@ -123,16 +123,25 @@ To prevent this behaviour, you can specify features as boot features.
...
 
 * `featuresRepositories` contains a list (comma-separated) of features repositories (features XML) URLs.
 * `featuresBoot` contains a list (comma-separated) of features to install at boot.
 
+========
+To remove features from the `featuresBoot` list in the `etc/org.apache.karaf.features.cfg` configuration file:
+
+. Navigate to `etc/org.apache.karaf.features.cfg`.
+. Remove the undesired feature.
+. Restart Fuse.
```
Comment (fpapon) [inline manual/src/main/asciidoc/user-guide/provisioning.adoc]: I think this line is wrong ;)

Comment (alex-hat-de-rojo) [reply manual/src/main/asciidoc/user-guide/provisioning.adoc]: Ah, yes, silly me. :)

**Comment by alex-hat-de-rojo** [comment]
Ready for review - @fpapon

**Comment by alex-hat-de-rojo** [comment]
Thanks @fpapon. Let me know what you think @jbonofre. :)

File: manual/src/main/asciidoc/user-guide/provisioning.adoc [hunk 2]
```diff
@@ -123,16 +123,25 @@ To prevent this behaviour, you can specify features as boot features.
...
 
-Apache Karaf features configuration is located in the `etc/org.apache.karaf.features.cfg` configuration file.
+The Apache Karaf features configuration is located in the `etc/org.apache.karaf.features.cfg` configuration file.
 
 This configuration file contains the two properties to use to define boot features:
 
 * `featuresRepositories` contains a list (comma-separated) of features repositories (features XML) URLs.
 * `featuresBoot` contains a list (comma-separated) of features to install at boot.
 
+========
```
Comment (jbonofre) [inline manual/src/main/asciidoc/user-guide/provisioning.adoc]: I think it's worth to mention that it's possible to stop Karaf, update `featuresBoot` and remove the data folder. It's another way to cleanup the `featuresBoot`.

**Comment by alex-hat-de-rojo** [comment]
Ready for final review.

**Comment by jbonofre** [comment]
@alex-hat-de-rojo thx ! I'm doing a new pass.

**Pull request merged**